gnomad/utils/annotations.py

```diff
@@ -1138,3 +1198,219 @@ def hemi_expr(
         # mt.GT[0] is alternate allele
         gt.is_haploid() & (sex_expr == male_str) & (gt[0] == 1),
     )
+
+
+def get_gks(
+    ht: hl.Table,
+    variant: str,
+    label_name: str,
+    label_version: str,
+    coverage_ht: hl.Table,
```

Comment:
```suggestion
    coverage_ht: hl.Table = None,
```

Reply:
making this none so it's not required to be passed in if a user just wants "vrs_only"